bindings/python/pymongoarrow/api.py

```diff
@@ -58,13 +58,13 @@ def find_arrow_all(collection, query, *, schema, **kwargs):
     context = PyMongoArrowContext.from_schema(
         schema, codec_options=collection.codec_options)
 
-    for opt in ('session', 'cursor_type', 'projection'):
+    for opt in ('session', 'cursor_type'):
```

Comment:
I believe with Python >=3.12 we _can_ support the session arg (added in https://jira.mongodb.org/browse/PYTHON-2389). Do you mind opening a new ticket to enable that feature here?